src/main/java/io/github/genomicdatainfrastructure/discovery/services/PackagesSearchResponseMapper.java

```diff
@@ -0,0 +1,129 @@
+// SPDX-FileCopyrightText: 2024 PNED G.I.E.
```

Comment:
Why we put mapper in /services folder, I would expect we have mappers folder? In this case it's fine because it's only one

Reply:
This is a fair question. There are two mappers there, and they are exclusivelly used by one service. They will never be reused by other classes. And they are classes just to simplify testing and reduce complexity of the service. Thus, it does not make sense to have a package `mappers` in the same level as `services`.

Anyway, my idea is to refactor a bit this code later on and move some components into modules and introduce some abstractions here and there to make this service reusable outside GDI context. This will happen in the following sprints, and hopefully we will come up with a better package convention.

Thanks for your concern.